SU-SWS/decanter
# DEC-381: Responsive flex classes with responsive gutters

# READY FOR REVIEW

# Summary
- Improves the developer experience in using the flex based classes so that the developer does not have to add the size for every breakpoint
- Breakpoint based flex size now has responsive behavior for all sizes above the one in use.

# Needed By (Date)
- End of the week

# Urgency
- Medium

# Steps to Test

1. Review code
2. Compile sass
3. Compile styleguide
4. Review core flex-box example for consistency with the existing version
5. Review flex-box layouts for issues

Test new behavior:
1. Create a new flexbox container and child element
2. Add different flexbox classes and review for expected behavior

# Affected Projects or Products
- Flex classes
- Flex layouts

# Associated Issues and/or People
- #381 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
